package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "mongoose-crud",
+  "version": "0.0.1",
+  "description": "Express middleware that creates all the CRUD endpoints for a mongoose model.",
+  "main": "lib/crud.js",
+  "scripts": {
+    "build": "babel --presets es2015 crud.js --out-dir lib --ignore test.js"
+  },
+  "keywords": [
+    "MongooseJS",
+    "MongoDB",
+    "Express",
+    "CRUD"
+  ],
+  "author": "Weston <Weston@blazingsolutions.net>",
+  "license": "MIT",
+  "dependencies": {
+    "babel-cli": "^6.24.1",
+    "babel-core": "^6.24.1",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
May want to look into `babel-preset-env` https://github.com/babel/babel/tree/master/experimental/babel-preset-env

It includes the 2015 preset as well as the newer ones. The gain with `env` is it automatically determines the Babel plugins you need based on your supported environments (configured in your babelrc)